src/app/lightning/payment-status-checker.ts

```diff
@@ -16,7 +16,7 @@ export const PaymentStatusChecker = ({ paymentRequest, lookupToken }) => {
 
   return {
     invoiceIsPaid: async (): Promise<boolean | RepositoryError> => {
-      const repo = InvoicesRepository()
+      const repo = WalletInvoicesRepository()
       const walletInvoice = await repo.findByPaymentHash(paymentHash)
       if (walletInvoice instanceof RepositoryError) return walletInvoice
```

Comment:
This should also use `isRepoError` or?

Reply:
I don't want to touch the code that arvin is working on until merge (base branch of this PR is his current working branch), I just removed the duplicated repo by his request